docker-compose.yml

```diff
@@ -31,6 +31,31 @@ services:
       HASURA_GRAPHQL_AUTH_HOOK: http://host.docker.internal:3000/api/hasura/auth
       ## HASURA_GRAPHQL_UNAUTHORIZED_ROLE: public
 
+
+  # for local s3 storage for images
+  localstack:
+    image: localstack/localstack:latest
+    hostname: s3
+    networks:
+      default:
+        aliases:
+          - s3.localhost.localstack.cloud
+          - s3-website.localhost.localstack.cloud
+          - coordinape.s3.localhost.localstack.cloud
+          - coordinape.s3-website.localhost.localstack.cloud
```

Comment:
oh I see, this was done here, is it needed to have so many aliases?

I imagine we could do this with all of the docker images, but probably not worth it, although more readable

Reply:
I was following examples from LocalStack, i think there is variance in how different versions of s3 refers to the host, older versions use folder paths and newer versions use subdomains. Hence the 2 versions. Then the two other hosts are the webserving hosts, so 4 total.